lib/zclient.c

```diff
@@ -435,6 +436,45 @@ enum zclient_send_status zclient_send_vrf_label(struct zclient *zclient,
 	return zclient_send_message(zclient);
 }
 
+enum zclient_send_status zclient_send_localsid(struct zclient *zclient,
+		const struct in6_addr *sid, ifindex_t oif,
+		enum seg6local_action_t action,
+		const struct seg6local_context *context)
+{
+	struct prefix_ipv6 p;
+	struct zapi_route api;
+	struct nexthop nh;
+
+	memset(&p, 0, sizeof(p));
```

Comment:
init locals with `{}` please

Reply:
I fixed all on my patch set.
Thank you.